fn main() {
    panic!("Test as a failure in build script.");
}
